sdk/eventhub/azure-eventhub/README.md

```diff
@@ -98,6 +94,12 @@ its partition; if there are multiple readers on the same partition, then they wi
 For more concepts and deeper discussion, see: [Event Hubs Features](https://docs.microsoft.com/azure/event-hubs/event-hubs-features).
 Also, the concepts for AMQP are well documented in [OASIS Advanced Messaging Queuing Protocol (AMQP) Version 1.0](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-overview-v1.0-os.html).
 
+### Thread safety
+
+We do not guarantee that the clients are thread-safe. We do not recommend reusing client instances across threads. It is up to the running application to use the clients in a thread-safe manner.
+
+The data model types, such as `EventDataBatch` are not thread-safe. They should not be shared across threads nor used concurrently with client methods.
```

Comment:
Again, I would be specific here, rather than using ambiguous terms like "such as". Give people an exact list of the objects they should be careful with.